modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/AddFormInstanceRecordMVCActionCommand.java

```diff
@@ -172,13 +160,56 @@ protected DDMForm getDDMForm(DDMFormInstance ddmFormInstance)
 		return ddmStructure.getDDMForm();
 	}
 
+	protected void saveFormInstanceRecord(
+			ActionRequest actionRequest, DDMFormInstance ddmFormInstance,
```

Comment:
@marcosapmf, this `actionRequest` is just used on the action `EditFormInstanceRecordMVCActionCommand`, can we avoid it? Ideally, we should find out a way to just share the logic that both action will need.